webapp/lib/app/nook/view.dart

```diff
@@ -1003,21 +1005,47 @@ class ConversationListPanelView {
     _conversationList = new LazyListViewModel(conversationListElement, onAddItemCallback: _onAddConversation);
     conversationListPanel.append(conversationListElement);
 
-    var panelFilters = new DivElement()
+    _panelFilters = new DivElement()
       ..classes.add('conversation-list-filters');
-    conversationListPanel.append(panelFilters);
+    conversationListPanel.append(_panelFilters);
 
-    conversationIdFilter = new ConversationIdFilter();
-    panelFilters.append(conversationIdFilter.conversationFilter);
+    var collapsePanelFilters = new DivElement()
+      ..append(SpanElement()..className = "fas fa-chevron-circle-down")
+      ..className = "conversation-filters-toggle"
+      ..onClick.listen((_) { _view.appController.command(UIAction.toggleFiltersPanel, ToggleFiltersPanelData(false)); });
+
+    conversationIdFilter = new ConversationIdFilter(collapsePanelFilters);
+    _panelFilters.append(conversationIdFilter.conversationFilter);
 
     conversationIncludeFilter = new ConversationIncludeFilter();
-    panelFilters.append(conversationIncludeFilter.conversationFilter);
+    _panelFilters.append(conversationIncludeFilter.conversationFilter);
 
     conversationExcludeFilter = new ConversationExcludeFilter();
-    panelFilters.append(conversationExcludeFilter.conversationFilter);
+    _panelFilters.append(conversationExcludeFilter.conversationFilter);
 
     conversationTurnsFilter = new ConversationTurnsFilter();
-    panelFilters.append(conversationTurnsFilter.conversationFilter);
+    _panelFilters.append(conversationTurnsFilter.conversationFilter);
+
+    _expandPanelFilters = new DivElement()
+      ..className = "conversation-filter-expand"
+      ..append(DivElement()
+          ..append(SpanElement()..className = "fas fa-filter")
+          ..append(SpanElement()..innerText = "Conversation filters")
+        )
+      ..append(DivElement()..append(SpanElement()..className = "fas fa-chevron-circle-up")..className = "conversation-filters-toggle")
+      ..onClick.listen((_) { _view.appController.command(UIAction.toggleFiltersPanel, ToggleFiltersPanelData(true)); })
+      ..hidden = true;
+    conversationListPanel.append(_expandPanelFilters);
+  }
+
+  void toggleFiltersMenu(bool show) {
+    if (!show) {
+      _panelFilters.hidden = true;
+      _expandPanelFilters.hidden = false;
+    } else {
+      _panelFilters.hidden = false;
+      _expandPanelFilters.hidden = true;
+    }
```

Comment:
nit - a more concise version:
```suggestion
    _panelFilters.hidden = !show;
    _expandPanelFilters.hidden = show;
```